portfolio/src/main/webapp/index.html

```diff
@@ -48,6 +48,14 @@ <h2>Socials</h2>
       <h2>Contact</h2>
         <p><b>Email:</b> aosmann7@gmail.com</p>
         <p><b>Phone:</b> 703-609-6892</p>
+      
+      <hr>
+      
+      <h2>Leave a Comment!</h2>
+        <form action="/data" method="POST">
+          <textarea name = "text-input" rows="25" cols="50"></textarea>
+          <br>
```

Comment:
For things like spacing between different objects, we usually prefer using css to style the margins instead of a <br>

Reply:
are you referring to the <br> tag here?